instructure/instructure-ui
# move from yarn to npm

INSTUI-3737

- CI breaks when `@netlify/esbuild-linux-64` is missing([support ticket #1](https://answers.netlify.com/t/deploy-fails-with-failed-to-find-package-netlify-esbuild-linux-64-on-the-file-system/97041), [support ticket #2](https://answers.netlify.com/t/github-actions-install-error-failed-to-find-package-netlify-esbuild-linux-64/91035)) -> works if it's manually installed from the workflow file

### Test plan

> Please note: this is a huge PR that touches a lot of our tooling so testing/reviewing should be done with extra care

- check out the PR and try running all of our usual scripts with npm (`npm install`, `npm run bootstrap`, `npm run dev`)
- try building the docs app
- try running tests
- check the docs: all (most) mentions of `yarn` should be replaced
- make a fresh clone and see if the usual scripts still run
- if anything else to test comes to your mind, please leave a comment about it
